nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -529,6 +529,52 @@ contract StakingEscrow is Issuer, IERC900History {
         emit ReStakeLocked(msg.sender, _lockReStakeUntilPeriod);
     }
 
+    /**
+    * @notice Enable `reStake` and lock this parameter even if parameter is locked
+    * @param _staker Staker address
+    * @param _info Staker structure
+    * @param _lockReStakeUntilPeriod Can't change `reStake` value until this period
+    */
+    function forceLockReStake(
+        address _staker,
+        StakerInfo storage _info,
+        uint16 _lockReStakeUntilPeriod
+    )
+        internal
+    {
+        // reset bit when `reStake` is already disabled
+        if (_info.flags.bitSet(RE_STAKE_DISABLED_INDEX) == true) {
+            _info.flags = _info.flags.toggleBit(RE_STAKE_DISABLED_INDEX);
+            emit ReStakeSet(_staker, true);
+        }
+        // lock `reStake` parameter if it's not locked or locked for too short duration
+        if (_lockReStakeUntilPeriod > _info.lockReStakeUntilPeriod) {
```

Comment:
It's not `>=`?

Reply:
if values are already equal then we can skip this step. For example, user locks restake by themself for long duration, which equal 6 month, then we don't want spend additional gas to change it